src/Symfony/Component/DependencyInjection/Dumper/YamlDumper.php

```diff
@@ -104,6 +104,11 @@ private function addService($id, $definition)
             $code .= sprintf("        synchronized: true\n");
         }
 
+        if ($definition->isDeprecated()) {
+            $code .= "        deprecated: true\n";
+            $code .= sprintf("        deprecation_message: %s\n", $definition->getDeprecationMessage('%service_id%'));
```

Comment:
It expects now the template instead, so this is a good remark (using `%service_id%` instead of the service's id)